helm/hpcc/values.yaml

```diff
@@ -82,8 +82,12 @@ global:
       #labels:
       #  mylabel: "4"
       type: LoadBalancer
-      
-      
+    # Ingress Network Polices will be created for following esp pods. The Network Policies will only take effect
+    # when a Kubernetes supported network polocy provider is ininstalled.
+    exposedServices: ["eclwatch", "eclqueries", "esdl-sandbox", "sql2ecl"]
+    # The following settings can be overwritted by ingress rules defined esp services/ingress
```

Comment:
>overwritted

overwritten

Reply:
> defined esp services/ingress

defined by the ingress section under each esp service.

Reply:
I could apply to other (non-esp) components too?
e.g. if someone wanted to expose the dafilesrv.service.
Perhaps drop the word 'esp' in comment.

Reply:
I only implemented the services which has "loadbalancer" now. Can we defer the other services later since each of them probably need treat differently.

I fixed the phrase but probably will remove "exposedServices" and move "ingress" to each esp service section.